src/app.js

```diff
@@ -320,6 +347,8 @@ var run = function () {
 
     currentFile = file
 
+    files.event.trigger('fileFocus', [file])
```

Comment:
After not passing the `files` object anymore this will anyway change.

Do you like the proposed api from above?

---
regarding the padding from above - which browser do you use? I can't reproduce this.